doc/modules/calibration.rst

```diff
@@ -74,10 +74,12 @@ by showing the number of samples in each predicted probability bin.
 
 .. currentmodule:: sklearn.linear_model
 
-:class:`LogisticRegression` returns well calibrated predictions by default as it has a
+:class:`LogisticRegression` is more likely to return well calibrated predictions by default as it has a
```

Comment:
"more likely" calls for a comparison: more likely than what?

I would also like to change the term "by default" to something better. Maybe "by itself", "without re-calibration, or "if well specified"